src/gate_detect/launch/video_test.launch

```diff
@@ -0,0 +1,35 @@
+<launch>
+       <!-- launch video stream -->
+     <include file="/home/cam/Jormungandr/external_pkg/video_stream_opencv/launch/camera.launch" >
+            <!-- node name and ros graph name -->
+            <arg name="camera_name" value="test_video" />
+            <!-- means video device 0, /dev/video0 -->
+            <arg name="video_stream_provider" value="/home/cam/Downloads/video-test/Approaching-90-degrees-from right.avi" />
+            <!-- node name and ros graph name -->
+                    <arg name="set_camera_fps" value="30"/>
+                    <!-- set buffer queue size of frame capturing to -->
+                    <arg name="buffer_queue_size" value="100" />
+                    <!-- throttling the querying of frames to -->
+                    <arg name="fps" value="30" />
+                    <!-- setting frame_id -->
+                    <arg name="frame_id" value="webcam" />
+                    <!-- camera info loading, take care as it needs the "file:///" at the start , e.g.:
+                    "file:///$(find your_camera_package)/config/your_camera.yaml" -->
+                    <arg name="camera_info_url" value="" />
+                    <!-- flip the image horizontally (mirror it) -->
+                    <arg name="flip_horizontal" value="false" />
+                    <!-- flip the image vertically -->
+                    <arg name="flip_vertical" value="false" />
+                    <!-- visualize on an image_view window the stream generated -->
+                    <arg name="visualize" value="false" />
+                        </include>
```

Comment:
Weird indenting here?

Reply:
Yeah I'm trying to figure out how to make clion format the xml's better, it doesn't do a great job with them

Reply:
yeah, that's unfortunately true. I think it's because they're indented with 2 spaces instead of the 4 clion defaults to.